dnf/cli/output.py

```diff
@@ -1628,7 +1628,13 @@ def historyListCmd(self, tids):
         fmt = "%s | %s | %s | %s | %s"
         if len(uids) == 1:
             name = _("Command line")
-            name_width = self.term.columns - 55 if self.term.columns > 79 else 24
+            if self.term.real_columns is None:
+                name_width = (
+                        24 if not transactions
+                        else max([len(t.cmdline or '') for t in transactions])
```

Comment:
Not sure why `else max([len(t.cmdline or '') for t in transactions])` is required but not required when `self.term.real_columns` is not none or `len(uids) != 1`? Please could you explain it? Also note in code is required if it is correct.